homeassistant/components/otbr/websocket_api.py

```diff
@@ -45,10 +45,13 @@ async def websocket_info(
         connection.send_error(msg["id"], "get_dataset_failed", str(exc))
         return
 
+    if dataset:
+        dataset = dataset.hex()
+
     connection.send_result(
         msg["id"],
         {
             "url": data.url,
-            "active_dataset_tlvs": dataset.hex(),
+            "active_dataset_tlvs": dataset,
```

Comment:
mypy should have caught this..
Can we somehow automatically default to strict typing for new integrations?

Reply:
We don't want to default to strict typing if it's a non-core integration. We don't want to make it tougher to contribute.